virt-v2v/cold/entrypoint.go

```diff
@@ -135,11 +135,11 @@ func customizeVM(source string, xmlFilePath string) error {
 			}
 		}
 
-		// RHEL
-		if strings.Contains(operatingSystem, "rhel") {
+		// Linux
+		if !strings.Contains(operatingSystem, "win") {
```

Comment:
Could we actually check for the `family == 'linux'` here.

osinfo-db has `family`, see: https://gitlab.com/libosinfo/osinfo-db/-/blob/main/data/os/fedoraproject.org/fedora-38.xml.in?ref_type=heads#L

Reply:
… with that approach we don't risk to be run on i.e. freebsd

Reply:
we use the xml output of virt-v2v, this information is missing, IMHO it's safe to try and run on any machine that is not identifying as windows, the script test for udev script and will not override if exist, and will fail gracefully if it can't find the network manager configuration scripts.